Core Content Expansion inc. new alien faction: the Xapleaux (#6439)
endless-sky/endless-sky: data/map.txt

			distance 307
			period 23.907

system Acrab
	pos -297.08377 146.935
	government Syndicate
	link Fomalhaut
	link Markab
	link Tabit
	asteroids "small rock" 40 4.224
	asteroids "large rock" 32 6.27
	asteroids "medium metal" 9 5.61
	asteroids "large metal" 4 4.158
	minables aluminum 3 10.167916
	minables iron 7 9.6900905
	fleet "Large Core Merchants" 800
	fleet "Small Core Pirates" 1200
	fleet "Small Core Merchants" 600
	habitable 151.25
	belt 1453
	object
		sprite star/m8
		period 10
	object

> 		sprite planet/rock5-b
> 		distance 255
> 		period 296.14716
> 	object
> 		sprite planet/browndwarf-t-rouge

Reviewer: In this extremely unusual case, the M8 may not actually be much brighter than the brown dwarf, so we might make an exception, but it still seems wrong to have it outside the orbit of this planet.

```suggestion
		sprite planet/browndwarf-t-rouge
		distance 255
		period 296.14716
	object
		sprite planet/rock5-b
```

Author: There is quite a bit of distance between them. The idea was that there's one planet that's close enough to the main star that its orbit can stably endure passing between the binary.
If the planet sprite needs to be changed though that's fair.

	attributes hot sheltered xapleaux
	link Ifk-groundissemour
	link Ink-learoumera-cehikli
	link Kiteg-hamlior-tock
	link Vik-benoureal
	asteroids "small rock" 1 9.75
	asteroids "medium rock" 1 9.295
	asteroids "large rock" 2 4.94
	asteroids "small metal" 17 6.76
	asteroids "medium metal" 40 3.445
	asteroids "large metal" 18 9.425
	minables aluminum 11 9.05446
	minables lead 3 6.1239
	minables titanium 3 10.5283
	haze _menu/haze-full
	fleet "Xapleaux Small Cargo" 1200
	fleet "Xapleaux Small Security" 1200
	fleet "Xapleaux Small Passenger" 1200
	habitable 589
	belt 1130
	trade Clothing 209
	trade Electronics 643
	trade Equipment 462
	trade Food 171
	trade "Heavy Metals" 1165
	trade Industrial 554
	trade "Luxury Goods" 1241
	trade Medical 708
	trade Metal 334
	trade Plastic 395
	hazard "Core Sheltered Heat" 1
	hazard "Core Shadow Passing" 9000
	object
		sprite star/k8
		period 10
	object Atke-babounriuer-taxerik
		sprite planet/cloud0-b
		distance 595
		period 287.25048
	object
		sprite planet/ice3
		distance 1320
		period 856.64366
	object
		sprite planet/gas6-b
		distance 2816.51
		period 4259.6788
		object
			sprite planet/rock0-b
			distance 397
			period 299.01046
	object

> 		sprite planet/browndwarf-l-rouge

Reviewer: K8 isn't very bright, but it should be enough to outshine this.  And this looks very bad with so many planets between the two.

```suggestion
		sprite planet/browndwarf-l
```

		distance 5347
		period 18399.514
		object
			sprite planet/europa
			distance 307
			period 254.45992

system Atlas
	pos -145.13889 174.86111
	government Syndicate
	link Achernar
	link "Al Dhanab"
	link Alpheratz
	link Meissa
	link Oneldiam
	asteroids "small rock" 17 3.069
	asteroids "medium rock" 5 5.3568
	asteroids "large rock" 8 4.5756
	asteroids "small metal" 29 4.5756
	asteroids "medium metal" 43 4.743
	asteroids "large metal" 1 2.9016
	minables silver 24 8.125609
	haze _menu/haze-133
	fleet "Large Core Pirates" 800
	fleet "Small Core Pirates" 1200
	fleet "Small Core Merchants" 600
	habitable 13107.2
	belt 1848
	object
		sprite star/b-supergiant
		period 10
	object
		sprite planet/ice3-b
		distance 633
		period 124.42155
		object
			sprite planet/desert4
			distance 192
			period 125.85303
	object
		sprite planet/gas14
		distance 1501
		period 454.31918
		object
			sprite planet/ice8-b
			distance 257
			period 155.74005
	object

> 		sprite planet/browndwarf-y-rouge

Reviewer: ```suggestion
		sprite planet/browndwarf-y
```

			distance 694
			period 68.5535

system "Beta Carinae"
	pos -84.250001 90.638889
	government Syndicate
	link "Beta Lyrae"
	link Mira
	asteroids "small rock" 16 5.2164
	asteroids "medium rock" 66 7.56
	asteroids "large rock" 129 4.914
	asteroids "small metal" 16 5.9724
	asteroids "medium metal" 5 5.8968
	asteroids "large metal" 21 5.67
	minables copper 15 11.250589
	minables aluminum 3 9.2244443
	minables iron 9 8.601015
	haze _menu/haze-133
	fleet "Large Core Pirates" 800
	fleet "Small Core Pirates" 1200
	fleet "Small Core Merchants" 600
	habitable 22579.2
	belt 1197
	object
		sprite star/a-supergiant
		period 10
	object
		sprite planet/gas7-b
		distance 810
		period 137.22026
		object
			sprite planet/rock20
			distance 278
			period 175.2136
	object

> 		sprite planet/browndwarf-y-rouge

Reviewer: ```suggestion
		sprite planet/browndwarf-y
```

system "Beta Lyrae"
	pos -120.44445 65.444447
	government Syndicate
	attributes core
	link "Beta Carinae"
	link Messiah
	link Ruchbah
	link Sheratan
	asteroids "small rock" 30 5.0232
	asteroids "medium rock" 34 5.2052
	asteroids "large rock" 11 1.82
	asteroids "small metal" 4 4.6228
	asteroids "medium metal" 199 2.4024
	asteroids "large metal" 46 4.4044
	minables iron 61 3.64072
	haze _menu/haze-133
	fleet "Small Core Merchants" 800
	fleet "Large Core Merchants" 1000
	fleet "Small Syndicate" 1600
	fleet "Large Syndicate" 2500
	fleet "Small Core Pirates" 2000
	fleet "Large Core Pirates" 4000
	habitable 2560
	belt 1130
	trade Clothing 310
	trade Electronics 750
	trade Equipment 673
	trade Food 174
	trade "Heavy Metals" 860
	trade Industrial 774
	trade "Luxury Goods" 1013
	trade Medical 671
	trade Metal 292
	trade Plastic 376
	object
		sprite star/o-supergiant
		distance 371
		period 357.29823
	object
		sprite star/f-supergiant
		distance 371
		period 357.29823
		offset 180
	object
		sprite planet/gas10
		distance 1162
		period 700.2195
	object
		sprite planet/jupiter-b
		distance 1884
		period 1445.5941
	object
		sprite planet/neptune-b
		distance 2544
		period 2268.3009
	object

> 		sprite planet/browndwarf-y-rouge

Reviewer: This one has got to be the worst.  A shadowless brown dwarf in a system with binary purple and white supergiants?  That's even more insane than the idea of this system.  And in human space no less!

```suggestion
		sprite planet/browndwarf-y
```

		period 342.094
		offset 312

system Electra
	pos -275.77778 73.111113
	government Syndicate
	link Botein
	link Iklil
	link Puppis
	asteroids "small rock" 9 3.4362
	asteroids "medium rock" 8 3.4362
	asteroids "large rock" 3 2.07
	asteroids "small metal" 5 2.898
	asteroids "medium metal" 6 2.4426
	asteroids "large metal" 3 2.1114
	minables silver 7 4.3545762
	minables copper 8 5.7354391
	haze _menu/haze-133
	fleet "Large Core Pirates" 800
	fleet "Small Core Pirates" 1200
	fleet "Small Core Merchants" 600
	habitable 2560
	belt 1607
	object
		sprite star/o-supergiant
		distance 358
		period 338.68389
	object
		sprite star/a-supergiant
		distance 358
		period 338.68389
		offset 180
	object
		sprite planet/gas10
		distance 976
		period 539.01362
	object
		sprite planet/gas9-b
		distance 1882
		period 1443.2928
		object
			sprite planet/ice7-b
			distance 265
			period 112.89805
	object
		sprite planet/gas12-b
		distance 3088
		period 3033.4787
	object

> 		sprite planet/browndwarf-y-rouge

Reviewer: ```suggestion
		sprite planet/browndwarf-y
```

		distance 2165.97
		period 1900.78

system Embe'kor
	pos 72.444444 178.77778
	government Korath
	attributes korath warm
	asteroids "small rock" 49 3.5672
	asteroids "medium rock" 4 3.2032
	asteroids "large rock" 46 3.458
	asteroids "small metal" 2 3.458
	asteroids "medium metal" 12 2.5844
	asteroids "large metal" 9 1.9656
	minables tungsten 5 5.213319
	haze _menu/haze-orange
	fleet "Korath Raid" 1600
	habitable 117964.8
	belt 1942
	trade Clothing 227
	trade Electronics 653
	trade Equipment 497
	trade Food 153
	trade "Heavy Metals" 1074
	trade Industrial 611
	trade "Luxury Goods" 1487
	trade Medical 587
	trade Metal 434
	trade Plastic 272
	hazard "Ember Waste Base Heat" 100
	hazard "Core Sheltered Heat" 16
	object
		sprite star/m-supergiant
		period 10
	object
		sprite planet/lava2
		distance 1092
		period 93.972922
	object
		sprite planet/jupiter-b
		distance 1840
		period 205.5396
	object
		sprite planet/gas17
		distance 3295
		period 492.55179
		object
			sprite planet/ice7
			distance 260
			period 109.71794
		object
			sprite planet/station2k
			distance 389
			period 200.78997
	object

> 		sprite planet/browndwarf-l-rouge

Reviewer: ```suggestion
		sprite planet/browndwarf-l
```

		distance 1751.6
		period 2523.74
	object
		sprite planet/gas17-b
		distance 2229.85
		period 3624.98

system Faerilloumier-iking-kicklissnika-puol
	pos 36.361067 154.935
	government Xapleaux
	link Dsik-geromil-jek
	asteroids "small rock" 16 3.3264
	asteroids "medium rock" 9 6.3504
	asteroids "large rock" 54 4.7088
	asteroids "small metal" 5 4.7952
	asteroids "medium metal" 23 2.5488
	asteroids "large metal" 6 5.4
	minables gold 13 8.16253
	minables iron 23 4.48663
	minables titanium 25 6.84244
	haze _menu/haze-cloudbright
	habitable 115184.45
	belt 1427
	object

> 		sprite star/black-hole

Reviewer 2: This should probably have the same effects that Sagittarius A* has, no? it was surprising that this black hole didn't suck me in and crush me into nothing

Reviewer 3: Sagittarus is the biggest BH in the galaxy, this is a random one so if it had an effect it should be way smaller

Reviewer 4: A stellar black hole is still a stellar black hole. As far as our ships are concerned, I'd say "only a few stellar masses" is about as deadly as "several thousand stellar masses." The lore is that our gravitational nullification systems that allow ships to launch from planets also provides strong protection from black holes. As in with it, they are "only" deadly over a short duration of exposure, as opposed to deadly instantaneously.

Author: I actually want to make this one as dangerous as Sag A is now, and then bump Sag A up to be *much* more dangerous.

Reviewer 3: > I actually want to make this one as dangerous as Sag A is now, and then bump Sag A up to be _much_ more dangerous.

once you fall into the gravity well of Sag A you're dead whatever you do pretty much, so how much dmg is done per second doesn't matter (it's less painful if more damage because it takes less long to die I think), but I the gravity pull could indeed be stronger

Reviewer 2: You absolutely can escape from Sag A* if you have a high enough thrust-mass ratio, it is not deadly unless you spend a prolonged amount of time sitting on top of it.

			distance 203
			period 17.525

system Feromansaath
	pos -125.33333 -138.66667
	government Uninhabited
	attributes korath
	asteroids "large rock" 1 4.34
	asteroids "medium metal" 3 3.162
	asteroids "large metal" 5 5.394
	minables titanium 2 9.0204489
	minables aluminum 1 7.5744573
	haze _menu/haze-133
	fleet "Large Quarg" 1000
	habitable 2163.2
	belt 1559
	trade Clothing 322
	trade Electronics 826
	trade Equipment 365
	trade Food 564
	trade "Heavy Metals" 1027
	trade Industrial 622
	trade "Luxury Goods" 1029
	trade Medical 484
	trade Metal 536
	trade Plastic 281
	object
		sprite star/o3
		period 10
	object
		sprite planet/rock12-b
		distance 525
		period 231.33195
		object
			sprite planet/luna-b
			distance 193
			period 101.35299
	object

> 		sprite planet/browndwarf-t-rouge

Reviewer: ```suggestion
		sprite planet/browndwarf-t
```

			distance 277
			period 13.5106

system Fulu
	pos -160.77778 39.888889
	government Syndicate
	link Ain
	link Messiah
	asteroids "large rock" 1 3.876
	asteroids "small metal" 7 2.958
	asteroids "medium metal" 172 4.233
	asteroids "large metal" 1 3.366
	minables tungsten 5 5.5887299
	minables aluminum 32 6.6162542
	minables platinum 8 5.1710371
	haze _menu/haze-133
	fleet "Large Core Pirates" 800
	fleet "Small Core Pirates" 1200
	fleet "Small Core Merchants" 600
	habitable 2560
	belt 1529
	object
		sprite planet/browndwarf-y-rouge
		distance 226
		period 169.87625
	object
		sprite planet/browndwarf-t-rouge
		distance 226
		period 169.87625
		offset 180
	object

> 		sprite planet/cloud5

Reviewer: Now this is actual proper usage of the `-rouge` variant.

	fleet "Large Saryd" 1200
	fleet Heliarch 900
	object
		sprite star/k5
		period 10
	object
		sprite planet/cloud6
		distance 256.01
		period 79.4787
	object "Shadow of Leaves"
		sprite planet/forest2-b
		distance 485.62
		period 207.639
	object
		sprite planet/rock11-b
		distance 689.03
		period 350.931
	object
		sprite planet/gas12-b
		distance 2352.59
		period 2214.03

system Ibvicklidit-fourelia
	pos 137.694 -127.065
	government Xapleaux
	link Ticska-hearaimiumer
	asteroids "small rock" 2 3.04
	asteroids "medium rock" 2 3.116
	asteroids "large rock" 1 4.674
	asteroids "medium metal" 1 4.788
	asteroids "large metal" 1 3.648
	haze _menu/haze-fullbright
	fleet "Xapleaux Small Security" 1200

> 	hazard "Ember Waste Base Heat" 1

Reviewer 2: ```suggestion
	hazard "Ember Waste Base Heat" 100
```
Otherwise, these systems which use EWBH invariably softlock every ship that enters them.

Reviewer 3: I spoke with him, it's intended for this space to be dangerous and requiring a special outfit to go there
but yeah it should be an event or smth that only lasts so long

Reviewer 5: Even then, this is probably too extreme.

10 or even 25 would probably be plenty for that.

"Dangerous" and "completely locks out anyone even if they try to prepare" are still different things ;)

Reviewer 2: I brought as many korath heat shunts as I could pack onto the ES, -19k heat at rest, and it was still no match for the heating
I think a frequency of 10 or 20 would do just fine as a very intense heat that (only) threatens to softlock you (but still leaves room for you to recover and escape) unless you're prepared

Reviewer 6: It might even make sense to have a dialog popup the first time the player enters the system to warn them of the heat (like what is done for the nanobots).

Author: Yeah this particular one is an accidental fail. XD

Reviewer 4: I'd personally prefer to have a gradual lead in. Have one or two systems leading up to it be quite hot (as in, push the average ship that normally sits at 25% heat up to 50%), then very hot (said average ship is ~75%), before getting into the severe heat.

I kind of disagree with making it completely impassable, though. If someone takes, say, a falcon, and strips out all the weapons to fill it with large heat shunts, it ought to be able to get through the initial periphery; although it could be something like the EW where it's not a  soft-lock, but there's waves of heat that paralyze the player semi-frequently...

Author: Yes well, there are notably new systems on the periphery with which to do that. 
Inside the actual defined area of Xapleaux space though, everything is already inside the periphery. There'll be variation within it, and it won't be impassable (unless you're running a hot stock Korath ship or something), but there are only two levels of heat in the core itself though and they're both hot.
A falcon that has stripped its weapons for heat shunts should be able to (once I have some brain space to address it) travel safely anywhere in the core. It won't be able to shoot at anything but it'll be able to fly.

			distance 123
			period 17.1483
	object
		sprite planet/rock15
		distance 1258.06
		period 543.125
	object
		sprite planet/gas17
		distance 2011.3
		period 1097.90
		object
			sprite planet/mercury-b
			distance 234
			period 14.441
		object
			sprite planet/desert4
			distance 351
			period 26.5299

system Krigec-svolurealuo-jik
	pos 155.694 -15.065
	government Xapleaux
	link Dygit-hourimaserai
	link Dygit-jarobouillier-cot
	asteroids "small rock" 2 2.106
	asteroids "medium rock" 15 1.456
	asteroids "large rock" 1 2.392
	asteroids "small metal" 1 2.314
	asteroids "medium metal" 2 2.34
	asteroids "large metal" 14 1.872
	minables tungsten 1 2.6314035
	minables silicon 1 3.2372957
	haze _menu/haze-full
	fleet "Xapleaux Small Cargo" 1200
	fleet "Xapleaux Small Security" 1200
	fleet "Xapleaux Small Passenger" 1200

> 	hazard "Ember Waste Base Heat" 1

Reviewer 2: ```suggestion
	hazard "Ember Waste Base Heat" 100
```
here too, otherwise it softlocks the player upon entering the system

			distance 403
			period 28.7798

system Poli'kor
	pos -26.888889 116.33333
	government Korath
	attributes korath warm
	asteroids "small rock" 2 2.079
	asteroids "medium rock" 18 2.5872
	asteroids "large rock" 7 1.6632
	asteroids "small metal" 38 2.6334
	asteroids "medium metal" 155 2.2407
	asteroids "large metal" 5 1.9173
	minables iron 48 2.87932
	minables titanium 16 2.87553
	minables uranium 13 3.05224
	haze _menu/haze-orange
	fleet "Korath Raid" 1600
	habitable 2560
	belt 1444
	trade Clothing 312
	trade Electronics 738
	trade Equipment 427
	trade Food 565
	trade "Heavy Metals" 1253
	trade Industrial 822
	trade "Luxury Goods" 1164
	trade Medical 471
	trade Metal 465
	trade Plastic 406
	hazard "Core Sheltered Heat" 16
	object

> 		sprite star/giant

Reviewer: The old giant is kept only for backwards compatibility, it's intended to be removed in the future.  Probably not a good idea to add them.

Can't say much better for the `-old` stars either, I'd rather rename `k5-old` to something like `m-subgiant` or `m5iv`, but that's further down the road if ever.

```suggestion
		sprite star/k5-old
```

		distance 159.5
		period 100.71891
	object
		sprite star/f-giant
		distance 159.5
		period 100.71891
		offset 180
	object
		sprite planet/rhea
		distance 522
		period 210.8292
	object
		sprite planet/gas10
		distance 1088
		period 634.40757
	object
		sprite planet/jupiter-b
		distance 2044
		period 1633.6023
	object
		sprite planet/neptune-b
		distance 2924
		period 2795.0595
	object

> 		sprite planet/browndwarf-y-rouge

Reviewer: ```suggestion
		sprite planet/browndwarf-y
```

			distance 416
			period 37.7101

system Sejuka
	pos -71.791722 -153.812
	government Uninhabited
	attributes korath
	asteroids "small rock" 13 2.6163
	asteroids "medium rock" 1 2.4871
	asteroids "large rock" 2 3.0362
	asteroids "large metal" 1 2.6809
	minables tungsten 1 3.1570134
	minables aluminum 1 3.0425517
	haze _menu/haze-133
	fleet "Large Quarg" 1000
	habitable 1641.25
	belt 1029
	trade Clothing 322
	trade Electronics 826
	trade Equipment 365
	trade Food 564
	trade "Heavy Metals" 1027
	trade Industrial 622
	trade "Luxury Goods" 1029
	trade Medical 484
	trade Metal 536
	trade Plastic 281
	object
		sprite star/k5
		distance 166.50724
		period 68.873863
		offset 180
	object

> 		sprite star/giant

Reviewer: The old giant is kept only for backwards compatibility, it's intended to be removed in the future.  Probably not a good idea to add them.

Can't say much better for the `-old` stars either, I'd rather rename `k5-old` to something like `m-subgiant` or `m5iv`, but that's further down the road if ever.

```suggestion
		sprite star/k5-old
```

		distance 316.26
		period 101.978
	object
		sprite planet/cloud2
		distance 876.62
		period 470.604
	object
		sprite planet/gas0
		distance 1759.83
		period 1338.58
		object
			sprite planet/desert4
			distance 254
			period 11.8633

system Vikinik-bounera
	pos 14.027703 -9.0649
	government Xapleaux
	attributes exposed hot xapleaux
	link Atch-werimoural-nikt
	link Crxst-xialourum
	link Icibvirk-gwolav-jek
	asteroids "large rock" 2 2.964
	asteroids "small metal" 28 2.8652
	asteroids "medium metal" 9 2.4206
	asteroids "large metal" 2 3.3098
	minables copper 1 5.54352
	minables tungsten 9 5.10205
	haze _menu/haze-fullbright
	fleet "Xapleaux Small Cargo" 1200
	fleet "Xapleaux Small Security" 1200
	fleet "Xapleaux Small Passenger" 1200
	fleet "Xapleaux Small Cargo" 1200
	fleet "Xapleaux Small Security" 1200
	fleet "Xapleaux Small Passenger" 1200
	habitable 233.28
	belt 1353
	trade Clothing 271
	trade Electronics 812
	trade Equipment 687
	trade Food 259
	trade "Heavy Metals" 650
	trade Industrial 795
	trade "Luxury Goods" 1164
	trade Medical 685
	trade Metal 280
	trade Plastic 443
	hazard "Core Exposed Heat" 1
	hazard "Core Heat Flare" 1200
	hazard "Core Shadow Passing" 21000
	object

> 		sprite star/m4

Reviewer: The M4 is kept only for backwards compatibility, it's intended to be removed in the future.  It has no high-dpi sprite IIRC and is out of pattern with the other stars.  Probably not a good idea to add them.

```suggestion
		sprite star/m3
```

		period 1975.32

system Xivik-soubiae-tock
	pos 56.805511 78.268333
	government Xapleaux
	attributes hot sheltered xapleaux
	link Rikt-pefaerilum
	asteroids "small rock" 5 5.796
	asteroids "medium rock" 33 2.562
	asteroids "large rock" 45 3.696
	asteroids "small metal" 7 2.646
	asteroids "medium metal" 61 2.604
	asteroids "large metal" 37 2.982
	minables lead 2 4.61282
	minables silver 2 4.71244
	haze _menu/haze-full
	fleet "Xapleaux Small Cargo" 1200
	fleet "Xapleaux Small Security" 1200
	fleet "Xapleaux Small Passenger" 1200
	habitable 4000
	belt 1270
	trade Clothing 272
	trade Electronics 869
	trade Equipment 707
	trade Food 385
	trade "Heavy Metals" 701
	trade Industrial 588
	trade "Luxury Goods" 1214
	trade Medical 577
	trade Metal 258
	trade Plastic 421
	hazard "Core Sheltered Heat" 1
	hazard "Core Shadow Passing" 9000
	object
		sprite star/o-supergiant
		distance 371
		period 357.29823
	object
		sprite star/f-supergiant
		distance 371
		period 357.29823
		offset 180
	object
		sprite planet/gas10
		distance 1162
		period 700.2195
	object
		sprite planet/jupiter-b
		distance 1884
		period 1445.5941
	object
		sprite planet/neptune-b
		distance 2544
		period 2268.3009
	object

> 		sprite planet/browndwarf-y-rouge

Reviewer: ```suggestion
		sprite planet/browndwarf-y
```

	outfitter "Coalition Basics"
	"required reputation" 15

planet Atch-werimoural-tinikt
	attributes farming xapleaux
	landscape land/fields3

> 	description `This world is possibly the most "normal" habitable world by galactic standards found anywhere in the core. With only a few small but extremely deep oceans and located in a darker, cooler region of the core, it almost has a weather cycle that could be termed familiar to outsiders. If Dygit-jarobouillier-cehicot feeds the Home Stretch, Atch-werimoural-tinikt feeds the the Waning Reach and the Quiet Arm, with the majority of its land area given over to agriculture or ecological products used to maintain the terraformed environments of some other Xapleaux worlds.`

Reviewer 7: ```suggestion
	description `This world is possibly the most "normal" habitable world by galactic standards found anywhere in the core. With only a few small but extremely deep oceans, along with its location in a darker, cooler region of the core, it almost has a weather cycle that could be termed familiar to outsiders. If Dygit-jarobouillier-cehicot feeds the Home Stretch, Atch-werimoural-tinikt feeds the the Waning Reach and the Quiet Arm, with the majority of its land area given over to agriculture or ecological products used to maintain the terraformed environments of some other Xapleaux worlds.`
```

	spaceport "The spaceport is a massive sprawling affair but is largely populated by plants, boxes and machines. The actual number of Xapleaux on this world is relatively low and there's no tourism to speak of, leading to a very business-like atmosphere with a high degree of automation."
	government Xapleaux

planet Atke-babounriuer-taxerik
	attributes urban xapleaux
	landscape land/mountain14-sfiera

> 	description "This world was once struck by an object of enormous mass and force, enough to cause the entire crust of the world to ripple violently. Only one relatively flat area of land exists on the surface, a crater surrounded on all sides by concentric rings of mountains which gradually decrease in height as they move away from the crater. Directly opposite the crater, on the other side of the world, a solid circle of mountains form a thicket of jagged peaks that reach almost to space in their center."

Reviewer 7: ```suggestion
	description "This world was once struck by an object of enormous mass and force, enough to cause the entire crust of the world to ripple violently. Only one relatively flat area of land exists on the surface: a crater surrounded on all sides by concentric rings of mountains which gradually decrease in height as they move away from the crater. Directly opposite the crater, on the other side of the world, a solid circle of mountains form a thicket of jagged peaks that reach almost to space in their center."
```

> 	spaceport "The spaceport, like almost all structures on this world, is built on the side of a mountain at the altitude most compatible with the majority of Xapleaux's needs. Despite the engineering challenges, the Xapleaux's conventional architecture remains at large, with buildings comprised of wide, flat floors layered atop each other in the usual, slightly uneven, pancake-stack style. Only here, these structures jut out over the valleys below like polypores (bracket or shelf fungi), sometimes far enough to give the illusion of floating above a cloudbank when the frequent dense fog fills the valley below."

Reviewer 7: `polypores (bracket or shelf fungi` sounds kind of clunky, so I replaced it with something that more players will probably be familiar with. However, I don't know if it still fits, so feel free to ignore this suggestion. 
```suggestion
	spaceport "The spaceport, like almost all structures on this world, is built on the side of a mountain at the altitude most compatible with the majority of the Xapleaux's needs. Despite the engineering challenges, the Xapleaux's conventional architecture remains at large, with buildings comprised of wide, flat floors layered atop each other in the usual, slightly uneven, pancake-stack style. Only here, these structures jut out over the valleys below like cliffs, sometimes far enough to give the illusion of floating above a cloudbank when the frequent dense fog fills the valley below."
```

Author: Took the typo, but I think the other change actually loses the meaning. There isn't really anything else out there that fits.
```suggestion
	spaceport "The spaceport, like almost all structures on this world, is built on the side of a mountain at the altitude most compatible with the majority of the Xapleaux's needs. Despite the engineering challenges, the Xapleaux's conventional architecture remains at large, with buildings comprised of wide, flat floors layered atop each other in the usual, slightly uneven, pancake-stack style. Only here, these structures jut out over the valleys below like polypores (bracket or shelf fungi), sometimes far enough to give the illusion of floating above a cloudbank when the frequent dense fog fills the valley below."
```

	description `	The solar collectors are mounted on frames about ten meters above the shifting sand dunes. In addition to turning to face the sun, they can be turned upside-down as necessary to dump off any sand that accumulates on them.`
	"required reputation" 15

planet Ceffig-haforiluour-repiker
	attributes tourism urban xapleaux
	landscape land/water6

> 	description "This large wet world produces very little of note, though other than importing some metals and electronics it's largely self-sufficient. It is the only link between the Waning Reach and the homeworld without traversing the full circumference of The Brilliance, or passing before the great eye of the cosmos; something the Xapleaux will not do unless quite desperate, or quite brave. As one of the safer worlds in their space though it is heavily populated, and home to the Montane Garden to which many make pilgrimages to consult with their elders."

Reviewer 7: ```suggestion
	description "This large wet world produces very little of note; other than importing some metals and electronics it's largely self-sufficient. It is the only link between the Waning Reach and the homeworld without traversing the full circumference of The Brilliance, or passing before the great eye of the cosmos; something the Xapleaux will not do unless quite desperate, or quite brave. It is one of the safer worlds in their space and as such it is heavily populated. It is also home to the Montane Garden to which many make pilgrimages to consult with their elders."
```

	description `	The solar collectors are mounted on frames about ten meters above the shifting sand dunes. In addition to turning to face the sun, they can be turned upside-down as necessary to dump off any sand that accumulates on them.`
	"required reputation" 15

planet Ceffig-haforiluour-repiker
	attributes tourism urban xapleaux
	landscape land/water6
	description "This large wet world produces very little of note; other than importing some metals and electronics it's largely self-sufficient. It is the only link between the Waning Reach and the homeworld without traversing the full circumference of The Brilliance, or passing before the great eye of the cosmos; something the Xapleaux will not do unless quite desperate, or quite brave. It is one of the safer worlds in their space and as such it is heavily populated. It is also home to the Montane Garden to which many make pilgrimages to consult with their elders."

> 	spaceport "For a world as populous as this one the spaceport is underwhelming. There is a decent amount of people coming and going, but for most cargo ships there's no reason to stop here. So many Xapleaux spaceports are so constrained by the practical needs of their economy though, that this one stands out as being quite unique. There is a focus on culture, arts, services, and experiences that just isn't present in the spaceports themselves on other worlds, making a visit an altogether pleasant experience."

Author: Needs an extra word removed for this
```suggestion
	spaceport "For a world as populous as this one the spaceport is underwhelming. There is a decent amount of Xapleaux coming and going, but for most cargo ships there's no reason to stop here. Many Xapleaux spaceports are constrained by the practical needs of their economy, but this one stands out as being quite unique. There is a focus on culture, arts, services, and experiences that just isn't present in the spaceports themselves on other worlds, making a visit an altogether pleasant experience."
```

	shipyard Kimek
	outfitter "Coalition Advanced"

planet Kiteg-hamlior-tancla
	attributes tourism urban xapleaux
	landscape land/dmottl2
	description "This quite dry and dusty world is dominated by chaparral ecosystems, and often as not rain is the result of smoke in the atmosphere instead of merely the accumulation of water vapor. The ground tremors frequently on account of its very large stellar neighbor: a moon nearly half its own mass. As a consequence Xapleaux structures are built well into the ground to rest upon bedrock instead of shifting soils, with an even greater preponderance of crystal skylights than normal to transmit the light of The Brilliance down into lower levels."

> 	spaceport `Plastics and medicines, medicinal plastics, plastics in medicines, and medicines in plastic. These are the exports of this world which has buried more vegetation than most worlds have ever grown. The Xapleaux here run the gamut of the more extremely bizarre looking variants, which dominate the population; so much so that the spaceport has multiple "burning" stations to accommodate the rarest variety of Xapleaux that can frequently be seen desperately waddling into the mirrored glass chamber and promptly catching fire. After a few moments of billowing flame they emerge, shaking off ash and looking much less burdened with the weight of their overgrown fronds burned away.`

Author: No, the point is that this is home to almost exclusively, just the most bizarre varieties.
```suggestion
	spaceport `Plastics and medicines, medicinal plastics, plastics in medicines, and medicines in plastic. These are the exports of this world which has buried more vegetation than most worlds have ever grown. The the full gamut of the uncommon and extremely bizarre looking variants of Xapleaux dominate the local population here; so much so that the spaceport has multiple "burning" stations to accommodate the rarest variety of Xapleaux that can frequently be seen desperately waddling into the mirrored glass chamber and promptly catching fire. After a few moments of billowing flame they emerge, shaking off ash and looking much less burdened with the weight of their overgrown fronds burned away.`
```

	asteroids "small rock" 30 5.0232
	asteroids "medium rock" 34 5.2052
	asteroids "large rock" 11 1.82
	asteroids "small metal" 4 4.6228
	asteroids "medium metal" 199 2.4024
	asteroids "large metal" 46 4.4044
	minables iron 61 3.64072
	haze _menu/haze-133
	fleet "Small Core Merchants" 800
	fleet "Large Core Merchants" 1000
	fleet "Small Syndicate" 1600
	fleet "Large Syndicate" 2500
	fleet "Small Core Pirates" 2000
	fleet "Large Core Pirates" 4000
	fleet "Human Miners" 2000
	habitable 861.25
	belt 1130
	trade Clothing 310
	trade Electronics 750
	trade Equipment 673
	trade Food 174
	trade "Heavy Metals" 860
	trade Industrial 774
	trade "Luxury Goods" 1013
	trade Medical 671
	trade Metal 292
	trade Plastic 376
	object
		sprite star/k8
		distance 85.63135
		period 54.596953
	object
		sprite star/a8
		distance 61.86865
		period 54.596953
		offset 180
	object
		sprite planet/ice6
		distance 333
		period 185.20245
	object
		sprite planet/tethys
		distance 621
		period 471.64776
	object
		sprite planet/gas10-b
		distance 1275
		period 1387.5405
	object
		sprite planet/gas16-b
		distance 2282
		period 3322.4147
		object
			sprite planet/miranda-b
			distance 230
			period 165.00746
	object

> 		sprite planet/sheragi_postverta

Reviewer 8: This definitely shouldn't be here.
